daemon/audit.c

```diff
@@ -652,11 +652,15 @@ audit_kernel_handle_log(int fd, UNUSED unsigned events, UNUSED event_io_t *io, v
 	char *buf = mem_new0(char, MAX_AUDIT_MESSAGE_LENGTH);
 	char *log_record = NULL;
 
+	if (events & EVENT_IO_EXCEPT) {
+		goto out;
```

Comment:
Would you consider any further steps necessary other than basically returning in case of an io-exection?